Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -383,7 +385,6 @@ void GcodeSuite::G28() {
                NUM_AXIS_LIST(              // Home each axis if needed or flagged
                  homeX = needX || parser.seen_test('X'),
                  homeY = needY || parser.seen_test('Y'),
-                 homeZZ = homeZ,
```

Comment:
This removal causes a compilation error, as the number of valid arguments in NUM_AXIS_LIST is one fewer than the number of axes, it's been reintroduced in 9ccb8c4.